### ISSUE TITLE
SyntaxWarning with Python3.8

### ISSUE THREAD

**Issue by iamaziz** [jrnl-org/jrnl#922]
See below:

```
/usr/local/Cellar/jrnl/2.2_1/libexec/lib/python3.8/site-packages/jrnl/time.py:60: SyntaxWarning: "is" with a literal. Did you mean "=="?
```

**Comment by tjstub** [comment]
Randomly jumping in, but it looks like it was fixed [here](https://github.com/jrnl-org/jrnl/commit/59624a49419b5f7880547bb1ac2d9ddb34f97768#diff-364aeefa827f15278f254308500037e8)

Updating should fix your issue!

**Comment by wren** [comment]
Closing as dupe of #878.

@tjstub Thanks for jumping in! We always appreciate community members' help. :)

@iamaziz The fix for this is in v2.4-beta, which is out right now. No new bugs for that build have been reported, so if this keeps up, it'll be released officially as v2.4 on Saturday.

**Comment by iamaziz** [comment]
Thanks
